com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/CobolWorkspaceService.java

```diff
@@ -17,18 +17,11 @@
 
 import com.broadcom.lsp.domain.cobol.event.api.EventObserver;
 import com.broadcom.lsp.domain.cobol.event.model.RequiredCopybookEvent;
-import org.eclipse.lsp4j.WorkspaceFolder;
+import org.eclipse.lsp4j.services.WorkspaceService;
 
 import java.io.IOException;
-import java.nio.file.Path;
-import java.util.List;
 
 public interface CobolWorkspaceService
```

Comment:
I think this entire interface should be also removed

Reply:
Agree, but @sergiuilie also have to integrate some functionalities here. So in order to avoid any merge conflict I will wait for sergiu merge and after that I will create an issue to get rid of this class and fix the related unit tests.
Let me know if is ok.